### PR TITLE
Fix site reg business license updated in overview

### PR DESCRIPTION
**Pull request by hogmann**
See ticket for steps to test
https://bcgovmoh.atlassian.net/browse/PRIME-2120

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/site-registration/pages/business-licence-page/business-licence-form-state.class.ts [hunk 1]
```diff
@@ -89,14 +90,21 @@ export class BusinessLicenceFormState extends AbstractFormState<BusinessLicenceF
     });
   }
 
+  public patchDocument(document: BusinessLicenceDocument): void {
+    if (!this.businessLicence) {
+      return;
+    }
+    this.businessLicence.businessLicenceDocument = document;
+  }
```
Comment (AMiKaH) [inline prime-angular-frontend/src/app/modules/site-registration/pages/business-licence-page/business-licence-form-state.class.ts:98]: can this be changed to:


```
  public patchDocument(document: BusinessLicenceDocument): void {
    if (this.businessLicence) {
      this.businessLicence.businessLicenceDocument = document;
    }
  }
```

Comment (hogmann) [reply prime-angular-frontend/src/app/modules/site-registration/pages/business-licence-page/business-licence-form-state.class.ts:98]: This is an alternate way to write it, and currently our other patch methods fail out so I continued that trend with this patch method.

**Review by AMiKaH — APPROVED** [review]
🚀

**Review by johnnyl6 — APPROVED** [review]
LGTM

**Pull request merged**